backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -770,31 +770,25 @@ public FutureExperimentRunDAO(
         InternalFuture.supplyAsync(
             () -> {
               final var localQueryContext = new QueryFilterContext();
-              localQueryContext.getConditions().add("experiment_run.deleted = :deleted");
+              localQueryContext.getConditions().add("deleted = :deleted");
               localQueryContext.getBinds().add(q -> q.bind("deleted", false));
 
               if (!request.getProjectId().isEmpty()) {
-                localQueryContext
-                    .getConditions()
-                    .add("experiment_run.project_id=:request_project_id");
+                localQueryContext.getConditions().add("project_id=:request_project_id");
```

Comment:
FWIW I prefer the original line spacing (multiple lines vs a single line) but that's just my preference.

Reply:
we have used google formatter rules and we have jenkins job as well for the same https://jenkins.dev.verta.ai/job/build/job/autoformat/job/modeldb-backend/653/console so it did by auto code formatter.